### PR TITLE
fix #73697 novelgo.id

### LINKED ISSUE #73697: novelgo.id
### Issue URL (Anti Adblock Script)
 [https://novelgo.id/the-novels-extra/chapter-0-the-story-within-the-novel/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fnovelgo.id%2Fthe-novels-extra%2Fchapter-0-the-story-within-the-novel%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.6 nightly 13
Browser: | Chrome
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (180), <br>Block Push API, <br>Block Location API
DNS filtering: | server: `https://dns-unfiltered.adguard.com/dns-query`<br>
Filters: | <b>Ad Blocking:</b><br/> EasyList, <br/>Peter Lowe's Blocklist, <br/>AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> EasyPrivacy, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> Adblock Warning Removal List, <br/>I don't care about cookies, <br/>EasyList Cookie List, <br/>AdGuard Annoyances<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist, <br/>Spam404, <br/>NoCoin Filter List<br/><br/><b>Language-specific:</b><br/> EasyList Italy
Custom filter subscriptions: | https://raw.githubusercontent.com/AdguardTeam/FiltersRegistry/master/filters/filter_15_DnsFilter/filter.txt

### PR DESCRIPTION
**Pull request by wout-codes**
#73697

Fixed Anti-Adblock and `Advertisement` word leftover.

### CODE REVIEW

**Comment by Alex-302** [comment]
I don't see BAB script here. Can you reproduce it now?

**Comment by wout-codes** [comment]
> I don't see BAB script here. Can you reproduce it now?

Yes @Alex-302 
image

**Review by Alex-302 — APPROVED** [review]

File: EnglishFilter/sections/specific.txt [hunk 1]
```diff
@@ -254,6 +254,7 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+novelgo.id##div[class^="code-block-label"]
```
Comment (Alex-302) [inline EnglishFilter/sections/specific.txt:257]: Wrong file. => foreign.txt => Indonesian

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/specific.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt]: Fixed @Alex-302

Comment (wout-codes) [reply EnglishFilter/sections/specific.txt]: @Alex-302 ?

**Comment by wout-codes** [comment]
@Alex-302 @zloyden Can anyone of you merge this? Approved 3 days ago but still not merged.

**Comment by zloyden** [comment]
> Approved 3 days ago but still not merged.

I guess @Alex-302 noticed it https://github.com/AdguardTeam/AdguardFilters/pull/73699#discussion_r570901529.
I'll take a look.

**Review by zloyden — APPROVED** [review]

**Pull request merged**